src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerExecutor.cs

```diff
@@ -127,7 +132,7 @@ internal class AnalyzerExecutor
                 onAnalyzerException: onAnalyzerException,
                 analyzerExceptionFilter: null,
                 analyzerManager: analyzerManager,
-                analyzerExecutionTimeMapOpt: null,
+                analyzerExecutionTimeMapOpt: CreateAnalyzerExecutionTimeMap(logExecutionTime),
```

Comment:
I would actually remove the parameter `logExecutionTime` from this method. None of the current callsites pass in an explicit value for this flag, and hence will always be false.